### PR TITLE
Integration of sign-in function for UI

### PR DESCRIPTION
**Pull request by garyukong**
I have created sign-in functionality for the UI, plus a couple of cosmetic tweaks.

The sign-in functionality also included updates to the DialogueAgent() class so that it stores the associated patient id each time an instance is created (which can then be passed on to the SQL DB in order to identify the patients)

### CODE REVIEW

File: reco_analysis/reco_analysis/app/credentials.yaml
Comment (mike-khor) [inline reco_analysis/reco_analysis/app/credentials.yaml:1]: I don't love that login credentials are in a file that is committed in repo, but also recognize that this is good enough for now until we pipe it to our db

File: reco_analysis/reco_analysis/app/credentials.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+credentials:
+  usernames:
+    "12305811":
```
Comment (mike-khor) [inline reco_analysis/reco_analysis/app/credentials.yaml]: is the patient ID related to MIMIC ID?

File: reco_analysis/reco_analysis/app/README.md
Comment (mike-khor) [inline reco_analysis/reco_analysis/app/README.md:1]: I'd maybe relabel the directory from `app` to `chatbot_app` to disambiguate from a potential `summarization_app`

**Review by mike-khor — APPROVED** [review]
LGTM!

File: reco_analysis/reco_analysis/app/credentials.yaml
Comment (garyukong) [reply reco_analysis/reco_analysis/app/credentials.yaml:1]: Yeah I agree. I think ideally we store the login credentials in a db.

File: reco_analysis/reco_analysis/app/credentials.yaml (hunk 1 above)
Comment (garyukong) [reply reco_analysis/reco_analysis/app/credentials.yaml]: This is the patient ID corresponding to the transcripts that Annie produced.

File: reco_analysis/reco_analysis/app/README.md
Comment (garyukong) [reply reco_analysis/reco_analysis/app/README.md:1]: I'll relabel it.

**Pull request merged**